spec/APNS.spec.js

```diff
@@ -1,68 +1,31 @@
-var APNS = require('../src/APNS');
+let APNS = require('../src/APNS');
 
 describe('APNS', () => {
 
-  it('can initialize with single cert', (done) => {
-    var args = {
-      cert: 'prodCert.pem',
-      key: 'prodKey.pem',
+  it('can initialize with cert', (done) => {
+    let args = {
+      cert: new Buffer('testCert'),
+      key: new Buffer('testKey'),
       production: true,
-      bundleId: 'bundleId'
+      topic: 'topic'
```

Comment:
I'd keep bundleId as the main option with topic being optional to maximize backwards compatibility. 

perhaps if using bundleId we could log a warning: `bundleId is deprecated, use topic instead`